streamalert/shared/alert_table.py

```diff
@@ -59,6 +59,10 @@ def _paginate(func, func_kwargs):
     def rule_names(self):
         """Find all of the distinct rule names in the table.
 
+        @deprecated
```

Comment:
is this still used internally? I'd say, if not, just 86 it

Reply:
It's used in a variety of places; as far as I can tell it's used in a lot of tests. My worry was it was used in a bunch of public methods, which other customers might be using... that said, I think it's a good idea cuz using this method is just not a good idea. I'll hunt down the references in a separate PR

Reply:
Separate note: "just 86 it" 🤣 

I watch too much food TV

Reply:
YOU or I watch too much!? I worked in a restaurant 😛

Reply:
I'm just surprised I understood the kitchen jargon
Too much Gordon Ramsey on Youtube
Yelling at people
ITS R AW
☠️

Reply:
LOL